### PR TITLE
chore: improve contributing

### PR DESCRIPTION
**Pull request by eliobischof**
- Improves README.md
- Improves CONTRIBUTING.md
- Adds and describes the option to develop against a local ZITADEL
- Automates setting up a local ZITADEL

### CODE REVIEW

File: CODE_OF_CONDUCT.md
Comment (eliobischof) [inline CODE_OF_CONDUCT.md:1]: This file feels arrogant to me and I don't think anybody cares about it :smile:

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,66 +1,71 @@
...
 
-### Pull Requests
+Please consider the following guidelines when creating a pull request.
 
-- The latest changes are always in `main`, so please make your Pull Request against that branch.
-- Pull Requests should be raised for any change
-- Pull Requests need approval of a ZITADEL core engineer @zitadel/engineers before merging
+- The latest changes are always in `main`, so please make your pull request against that branch.
+- pull requests should be raised for any change
+- üull requests need approval of a ZITADEL core engineer @zitadel/engineers before merging
```
Comment (peintnermax) [inline CONTRIBUTING.md:27]:
```suggestion
- Pull requests need approval of a ZITADEL core engineer @zitadel/engineers before merging
```

**Review by peintnermax — CHANGES_REQUESTED** [review]
We should backup the current `.env.local` before we overwrite the variables.. Rest works nicely 👍

**Comment by eliobischof** [comment]
> We should backup the current `.env.local` before we overwrite the variables.. Rest works nicely +1

Automated it https://github.com/zitadel/typescript/pull/20/commits/5748aab4d4b342e6a9d0efe2972343555736f52d

**Review by peintnermax — APPROVED** [review]

**Pull request merged**